jerry-core/debugger/debugger.h

```diff
@@ -354,6 +354,7 @@ typedef struct
 typedef struct
 {
   uint8_t type; /**< type of the message */
+  uint8_t total_frames[sizeof (uint32_t)]; /**< number of total frames */
   jerry_debugger_frame_t frames[]; /**< frames */
```

Comment:
Is this part of all backtraces?

Reply:
Yes indeed it is part of every backtrace request. The total frames is at the beginning, so only the first transmitted message will contain this data.

Reply:
I would rather use a separate message type for sending this data. It makes the client side quite complex.